GitUI/UserControls/FileStatusDiffCalculator.cs

```diff
@@ -88,6 +91,34 @@ public IReadOnlyList<FileStatusWithDescription> Reload()
                     }
                 }
 
+                // Show diff with base regarding default branch of default remote
+                IReadOnlyList<string> remotes = module.GetRemoteNames();
+                if (remotes.Count == 0)
+                {
+                    return fileStatusDescs;
+                }
+
+                string defaultRemote = remotes.FirstOrDefault(r => r is "origin" or "upstream") ?? remotes[0];
+                GitArgumentBuilder args = new("symbolic-ref") { $"{GitRefName.RefsRemotesPrefix}{defaultRemote}/HEAD" };
```

Comment:
I do not believe that HEAD for the remotes are tracked by default, no info that I see at least.

Reply:
It shall and does return the repo's default branch. But yes, HEAD cannot be resolved for many of my repos.